Marlin/src/gcode/probe/M851.cpp

```diff
@@ -74,7 +78,12 @@ void GcodeSuite::M851() {
   }
 
   // Save the new offsets
-  if (ok) probe_offset = offs;
+  if (ok) {
+    probe_offset = offs;
+    #if ENABLED(DELTA)
+      delta_calibration_radius = DELTA_CALIBRATION_RADIUS;
```

Comment:
If the user has used `M665 B` to set a more correct radius, this will override that setting without informing the user, and may lead to unexpected behavior.